### ISSUE TITLE
Telegram don't find libbrotlidec.so.1 with snap version

### ISSUE THREAD

**Issue by giacomogroppi** [telegramdesktop/tdesktop#25164]
### Steps to reproduce

install telegram-desktop with snap (channel edge)

### Expected behaviour

open...

### Actual behaviour

the application does not start. Trying to open it from the terminal, the message is printed:
telegram-desktop: error while loading shared libraries: libbrotlidec.so.1: cannot open shared object file: No such file or directory

### Operating system

ubuntu 22.04

### Version of Telegram Desktop

4.2.4-5-g13103ad0e

### Installation source

Snap

**Comment by ilya-fedin** [comment]
Something really weird happens here... Only libjxl's copy of brotli is excluded from staging as it conflicts with the Ubuntu-provided brotli, but it looks like snapcraft removed the library from all the parts... It is either a bug of snapcraft or I don't understand how it's supposed to work...

**Comment by ilya-fedin** [comment]
Another interesting thing is it present in the stable build while there were no changes to snapcraft.yaml. So apparently some broken snapcraft update was released recently...

**Comment by ilya-fedin** [comment]
It seems there were no snapcraft releases since beginning of september... I have no idea why it has broke then :(

**Comment by dmitrygribenchuk** [comment]
Hello. Have the same issue

**Comment by ilya-fedin** [comment]
No need to write posts like 'same issue' if you folks don't want the issue to be locked

**Comment by hideuk** [comment]
same issue on Ubuntu 22.04 for snap edge (installing stable rn)

**Comment by vsevolod** [comment]
One more point:
I have this library at `/snap/telegram-desktop/4266/usr/lib/x86_64-linux-gnu/libbrotlidec.so.1`
But snap couldn't find it.

```bash
$ snap --version
snap    2.57.2
snapd   2.57.2
series  16
ubuntu  22.04
kernel  5.15.0-48-generic
```

I switched snap through Ubuntu-Software and installed Telegram again. It helps
![image](https://user-images.githubusercontent.com/110316/194044588-e9d1849a-2d1d-48a5-9ccc-6531f6e8b947.png)

**Comment by RudnievV** [comment]
> I switched snap through Ubuntu-Software and installed Telegram again. It helps ![image](https://user-images.githubusercontent.com/110316/194044588-e9d1849a-2d1d-48a5-9ccc-6531f6e8b947.png)

Thank you! It works for me!

**Comment by PeterZhizhin** [comment]
Here is a workaround that worked for me (switching to a stable version):
```
sudo snap refresh telegram-desktop --channel=latest/stable
```

**Comment by dmitrygribenchuk** [comment]
Fixed with update 4.2.4-7-g8f1a8f909 2022-10-06

**Comment by ilya-fedin** [comment]
Wow, it broke on itself and fixed on itself
